index.html

```diff
@@ -329,8 +329,8 @@ <h2>Introduction</h2>
       <p> 
 In other words, the
 controller document contains the necessary information for proving
-cryptographically that specific actions were taken by the
-controller of an identifier, including material for
+that specific actions were taken by the controller of an identifier,
```

Comment:
that either specific actions were taken by the, or for communicating with, the controller of an identifier